cypress/integration/mainSidebar.cy.js

```diff
@@ -1,9 +1,17 @@
+import {
+    ACCESSORY_PANEL_DEFAULT_WIDTH,
+    ACCESSORY_PANEL_MIN_WIDTH,
+    ACCESSORY_PANEL_MIN_PX_AT_END,
+    PRIMARY_PANEL_WIDTH,
+    ARROW_LEFT_KEY,
+    ARROW_RIGHT_KEY,
+} from '../../src/modules/accessoryPanelConstants.js'
```

Comment:
All these constants were moved to a separate file so they could also be imported into the e2e test files without causing transpilation errors